packages/types/surveys/types.ts

```diff
@@ -1387,3 +1387,14 @@ export const ZSurveyRecallItem = z.object({
 });
 
 export type TSurveyRecallItem = z.infer<typeof ZSurveyRecallItem>;
+
+export const CopySurveyFormValidation = z.object({
```

Comment:
Please follow naming convention. A zod type in surveys file should start with "ZSurvey"